libs/app-utils/models/src/lib/msc-harmony.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import {
+  AppFileSystemGuardOptions,
+  AppFileSystemScope,
+} from "./app-file-system"
+
+export const MSC_HARMONY_SCOPE: AppFileSystemScope = "userData" as const
+export const MSC_HARMONY_SCOPE_CATALOG_DIR = "msc-harmony" as const
+
+export const getMscHarmonyLocation = (
+  fileName?: string
+): AppFileSystemGuardOptions => {
+  return {
+    scopeRelativePath: fileName
+      ? [MSC_HARMONY_SCOPE_CATALOG_DIR, fileName]
+      : MSC_HARMONY_SCOPE_CATALOG_DIR,
+    scope: MSC_HARMONY_SCOPE,
+  }
+}
```

Comment:
This should not be a part of the `app-utils` library. It's a highly specific part of code, meant for the Harmony device in MSC mode only. Please move it to one of the libraries from the `devices/harmony-msc/` directory.

Reply:
https://github.com/mudita/mudita-center/pull/2750/commits/b1ea1ce01533a7e1eaa1cfd3e2550325b9bbf9e2